### PR TITLE
fix: hud styles

### PR DESCRIPTION
**Pull request by mollybee**

### CODE REVIEW

**Review by ethankarp — APPROVED** [review]
Wow that's a simple fix haha. I wonder why that Hud import was removed in the first place.

**Review by rhiannonstanford — APPROVED** [review]
Thanks for adding that import statement back in, restoring the progress bar on child/dashboard.

**Review by artofmayhem — APPROVED** [review]
simple and done. nice catch on that error

File: src/styles/less/index.less [hunk 1]
```diff
@@ -8,6 +8,7 @@
 @import 'ChildDashboard';
 @import 'Header';
 @import 'HelpPage';
+@import 'Hud';
```
Comment (Julianisrel) [inline src/styles/less/index.less:11]: Good Add!

**Pull request merged**